dll/win32/riched20/txtsrv.c

```diff
@@ -31,6 +34,12 @@
 #include "wine/debug.h"
 #include "editstr.h"
 
+#ifdef __REACTOS__
+#ifdef _MSC_VER
+#define __thiscall __stdcall
```

Comment:
This seems weird?

Reply:
Ya, this is redundant now that asm.h is included, I'll remove it.